protocol-methods.rst

```diff
@@ -2,6 +2,149 @@
  Protocol Methods
 ==================
 
+blockchain.address.get_balance
+==============================
+
+Return the confirmed and unconfirmed balances of a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.get_balance(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  * *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
+
+**Result**
+
+  See :func:`blockchain.scripthash.get_balance`.
+
+blockchain.address.get_history
+==============================
+
+Return the confirmed and unconfirmed history of a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.get_history(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  * *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
+
+**Result**
+
+  As for :func:`blockchain.scripthash.get_history`.
+
+blockchain.address.get_mempool
+==============================
+
+Return the unconfirmed transactions of a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.get_mempool(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  * *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
+
+**Result**
+
+  As for :func:`blockchain.scripthash.get_mempool`.
+
+blockchain.address.listunspent
+==============================
+
+Return an ordered list of UTXOs sent to a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.listunspent(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  * *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
+
+**Result**
+
+  As for :func:`blockchain.scripthash.listunspent`.
+
+blockchain.address.subscribe
+============================
+
+Subscribe to a Bitcoin Cash address.
+
+**Signature**
+
+  .. function:: blockchain.address.subscribe(address)
+  .. deprecated:: 1.2 removed in version 1.3, re-added in version 1.4.3
+
+  *address*
+
+    The address as a Cash Address (with or without prefix) or a Base58 string.
+
+**Result**
+
+  The :ref:`status <status>` of the address.
+
+**Notifications**
+
+  As this is a subcription, the client will receive a notification
+  when the :ref:`status <status>` of the address changes.  Its
+  signature is
+
+  .. function:: blockchain.address.subscribe(address, status)
+
+  .. note:: The address returned back to the client when notifying of status
+            changes will be in the same encoding and syle as was provided when
+            subscribing. In effect, a whitespace-stripped version of the address
+            string that the client provided will be sent back to the client when
+            notifying, in order to make it easier for clients to track the
+            notification.
```

Comment:
This seems necessarily hard to implement. Can't we just make this lower case representation of cashaddr with prefix? No ambiguity.

Reply:
Why be so restrictive though?  I see that as needlessly annoying... why?

Reply:
OK